src/main/java/com/ncc/aif/ValidateAIF.java

```diff
@@ -42,9 +50,12 @@
     private static final String AO_EVENTS_RESNAME = ONT_ROOT + "EventOntology";
     private static final String AO_RELATIONS_RESNAME = ONT_ROOT + "RelationOntology";
 
+    public static final String DOMAIN_MODEL_PATH = System.getProperty("java.io.tmpdir") + "/diskbased-models/domainModel";
```

Comment:
I wanted to limit the number of temp directories created, especially on Windows, because they are ~200 MB each and generally don't get cleaned up at the end of the run (even on Linux).  So I name them incrementally and delete them if they're there first (basically reusing the directory).  If you're running X disk-based validations at a time, it will create domainModel1 through domainModelX.

Reply:
I see. More control over the directory name.

Reply:
Right.  At one point I just nuked the domainModels subdir before I started, but obviously that didn't work well if you had multiple validations going on.  I guess I could check for directories named `diskModel-*` and delete/reuse them if they're not busy.

Reply:
Nah, this works. I just didn't see why you wanted to do it this way. I'm good with this approach for tmp files.